ios/Example/Tests/SecureEnclaveDigitalSignatureKeyManagerTests.swift

```diff
@@ -7,12 +7,15 @@ import Trifle
 import XCTest
 
 final class SecureEnclaveDigitalSignatureKeyManagerTests: XCTestCase {
-    lazy var manager = SecureEnclaveDigitalSignatureKeyManager(
-        tag: "app.cash.trifle.s2dk.keys.digital_signature"
-    )
-    lazy var otherManager = SecureEnclaveDigitalSignatureKeyManager(
-        tag: "com.squareup.trifle.s2dk.keys.digital_signature"
-    )
+    var manager: SecureEnclaveDigitalSignatureKeyManager!
```

Comment:
hmm any reason why we're changing this from a lazy initialization

Reply:
Since `SecureEnclaveDigitalSignatureKeyManager` can throw, we have to initialize in setUp() but lazy requires instantiation at the definition.
So had to remove lazy.